### DISCUSSION TITLE
✨ Zabbix

### DISCUSSION THREAD

**Discussion by JigSawFr** [runtipi/runtipi-appstore#2194] [Requests]
### Description

Real-time monitoring of IT components and services, such as networks, servers, VMs, applications and the cloud.

### Official Website

https://www.zabbix.com/

### Documentation link

https://www.zabbix.com/documentation/6.2/en/manual/installation/containers

### Last application release & date

6.4.10 (13/12/2023)

### Application license

GPL-2.0 license

### Source code repository link

https://github.com/zabbix/zabbix

### Docker image link

https://github.com/orgs/zabbix/packages?repo_name=zabbix-docker

### Please confirm the following

- [X] I confirm that I've put the app name in the title of the discussion
- [X] I confirm that the application is actively maintained.
- [X] I confirm that the docker image of the application is actively maintained.
- [X] I confirm that I have indeed checked that no similar request for this application already exists

### Gathering crowds

Hey folks!

Please upvote 👍 this discussion to show your interest in this request! 

Thanks ⛺

**Comment by JigSawFr** [comment]
Originally requested by @Kira-stuff

**Comment by jasonpearce** [comment]
It looks like the Docker files are now located here:

https://github.com/zabbix/zabbix-docker

For a full list of their container images, see:

https://www.zabbix.com/container_images
